addons/binding/pom.xml

```diff
@@ -48,6 +48,8 @@
     <module>org.openhab.binding.max.test</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).